.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Configuration for https://probot.github.io/apps/stale
```

Comment:
Do you think this bot will be useful here? I'm wondering if we should avoid using it for now, or use what we have in aragon/aragon (i.e. only enabled for PRs).

Reply:
Replaced the entire stale.yml with what you have in aragon/aragon. Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474